### PR TITLE
doc: add more extensive blobstore debugging tips

### PR DESCRIPTION
**Pull request by slimsag**
So far only 3 customers have run into a problem with blobstore; one was a case of not upgrading properly, another was unrelated to blobstore entirely (`HTTP_PROXY` issues), and another is ongoing.

This PR adds some docs that would've been helpful in those situations to give more confidence that blobstore is working as intended, help folks debug other unrelated issues that might initially appear as a blobstore issue, etc.

## Test plan

Docs only.

### CODE REVIEW

**Comment by slimsag** [comment]
cc @Galindo-J @Rhia2 just letting you know I'm adding these to our docs, hopefully these will be helpful in the future for debugging blobstore-related issues!

File: doc/admin/how-to/blobstore_debugging.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+This page provides more tips on debugging why blobstore may not be working properly.
+
+**Please feel free to contact support@sourcegraph.com** if you are encountering any issues and we can help you work through debugging steps.
+
+## Can I disable blobstore?
+
+Today, Sourcegraph uses blobstore for storing precise code intel (LSIF/SCIP) uploads. You can also configure Sourcegraph to use [S3 or GCS for object storage](../external-services/object_storage.md) if you prefer.
+
+In the near future, other SOurcegraph features like Batch Changes may also rely on object storage and so, if possible, it's best to make sure it is working.
```
Comment (Galindo-J) [inline doc/admin/how-to/blobstore_debugging.md:13]: Line 13: SOurcegraph -> Sourcegraph

File: doc/admin/how-to/blobstore_debugging.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+You should see a 200 response like:
+
+```
+HTTP/1.1 200 OK
+Date: Mon, 09 Jan 2023 20:47:32 GMT
+x-amz-request-id: 4442587FB7D0A2F9
+Content-Type: application/xml;charset=utf-8
+Transfer-Encoding: chunked
+Server: Jetty(11.0.11)
+```
```
Comment (Galindo-J) [inline doc/admin/how-to/blobstore_debugging.md:82]: Just to confirm, once they get to this part of the debugging guide, if they don't get a 200 response, are there steps we can provide them (can be added later), or is this where support steps in?

File: doc/admin/how-to/blobstore_debugging.md (hunk 1 above)
Comment (slimsag) [inline doc/admin/how-to/blobstore_debugging.md:13]:
```suggestion
In the near future, other Sourcegraph features like Batch Changes may also rely on object storage and so, if possible, it's best to make sure it is working.
```

File: doc/admin/how-to/blobstore_debugging.md (hunk 2 above)
Comment (slimsag) [reply doc/admin/how-to/blobstore_debugging.md:82]: I'm not sure what we would suggest next aside from contact us. If this happens it means you've done something quite strange with your deployment, e.g. you've really messed up networking somehow and containers cannot talk to each-other. These types of issues are super hard and require a lot more guesswork and wouldn't really be blobstore specific.

We could maybe link to a more general "how to debug if two containers cannot talk to eachother" doc - that'd be nice to have, though I don't have time to write it right now.

**Review by Galindo-J — APPROVED** [review]

**Pull request merged**